dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogConfigBuilderTest.kt

```diff
@@ -581,4 +583,67 @@ class DatadogConfigBuilderTest {
                 )
             )
     }
+
+    @Test
+    fun `builder with track gestures enabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .trackGestures()
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackGestures = true
+                )
+            )
+    }
+
+    @Test
+    fun `builder with track activities as screens enabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .trackActivitiesAsScreens()
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackActivitiesAsScreens = true
+                )
+            )
+    }
+
+    @Test
+    fun `builder with track fragments as screens enabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .trackFragmentsAsScreens()
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackFragmentsAsScreens = true
+                )
+            )
+    }
```

Comment:
Don't we miss tests for default values of `trackGestures`, `trackActivitiesAsScreens` and `trackFragmentsAsScreens`?

Reply:
Good catch ;)

Reply:
Done ;)